degiro_connector/trading/actions/action_update_order.py

```diff
@@ -120,11 +121,15 @@ def update_order(
         try:
             response_raw = session.send(prepped, verify=False)
             response_raw.raise_for_status()
+            response_dict = response_raw.json()
```

Comment:
I would remove this line.

Since it's not used.

Plus I am not sure that the API will return a `json` on success.

Reply:
:+1: